### PR TITLE
Bootstrap form theme and checkboxes

### PR DESCRIPTION
**Pull request by anacicconi**
| Q | A |
| --- | --- |
| Doc fix? | yes |
| New docs? | no |
| Applies to | 2.6 and greater |
| Fixed tickets | #4906 |

### CODE REVIEW

File: cookbook/form/form_customization.rst [hunk 1]
```diff
@@ -114,6 +114,13 @@ fragment needed to render every part of a form:
   but the CSS classes applied are the ones used to display the forms horizontally
   (i.e. the label and the widget in the same row).
 
+.. caution::
+
+  If you use the Bootstrap form theme and want to render the parts of 
+  a checkbox field individually, do not use the Twig function {{ form_label() }}. 
+  Otherwise, it will render nothing.
```
Comment (WouterJ) [inline cookbook/form/form_customization.rst]: this indentation has to be changed from 2 spaces to 4 spaces

File: cookbook/form/form_customization.rst [hunk 2]
```diff
@@ -114,6 +114,13 @@ fragment needed to render every part of a form:
   but the CSS classes applied are the ones used to display the forms horizontally
   (i.e. the label and the widget in the same row).
 
+.. caution::
+
+  If you use the Bootstrap form theme and want to render the parts of 
+  a checkbox field individually, do not use the Twig function {{ form_label() }}.
```
Comment (WouterJ) [inline cookbook/form/form_customization.rst]: You should put the code between double backticks, to make it an inline code block.

File: cookbook/form/form_customization.rst [hunk 3]
```diff
@@ -114,6 +114,13 @@ fragment needed to render every part of a form:
   but the CSS classes applied are the ones used to display the forms horizontally
   (i.e. the label and the widget in the same row).
 
+.. caution::
```
Comment (WouterJ) [inline cookbook/form/form_customization.rst]: I think "note" is better than "caution" here?

**Comment by WouterJ** [comment]
Hi @anacicconi! I've picked your commit, applied my comments and created a new pull request: https://github.com/symfony/symfony-docs/pull/5460 I'll now close it in favor of this new PR. Thank you for starting this! (of course, if you have comments on my changes, feel free to comment)

**Comment by anacicconi** [comment]
Hi @WouterJ, thanks for improving it ;)

**Pull request closed without merging**